docs/installation.rst

```diff
@@ -211,22 +211,25 @@ you can configure the nodes in very different ones to enable more
 aggressive/optimized LRU.
 
 That said, if you're running a small install you can probably get away
-with just setting up the defaults::
-
-    SENTRY_REDIS_OPTIONS = {
-        'hosts': {
-            0: {
-                'host': '127.0.0.1',
-                'port': 6379,
-                'timeout': 3,
-                #'password': 'redis auth password'
-            }
-        }
-    }
+with just setting up the defaults in ``config.yml``::
+
+    redis.clusters:
```

Comment:
I'd personally condense this down to 2 space indents. It'd make it a bit more readable imo since you're only showing one deeply nested object.